cfpb/consumerfinance.gov
# Archiving: hide post preview images for archived pages

As part of the initial archiving implementation, we grayscaled images on archived blog / newsroom / reports pages but did not change the appearance of preview images for archived pages. This PR hides those images for archived pages.



## Additions

- Updates post preview organism to exclude images for archived posts



## How to test this PR

1. Check out branch
2. Navigate to [blog](http://localhost:8000/about-us/blog/) and verify that
    - post preview images don't show for archived posts
    - post preview images still appear for non-archived posts


## Screenshots

### Blog filtered to archived posts on production

<img width="925" alt="Screen Shot 2020-12-07 at 9 19 00 AM" src="https://user-images.githubusercontent.com/778171/101382987-70345280-386d-11eb-9f5f-c07dac2e743a.png">

### Archived posts locally

<img width="968" alt="Screen Shot 2020-12-07 at 9 19 09 AM" src="https://user-images.githubusercontent.com/778171/101382998-732f4300-386d-11eb-99cc-3864ad1e5498.png">


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
